Fronter/Fronter.depend

```diff
@@ -0,0 +1,3779 @@
+# depslib dependency file v1.0
```

Comment:
Is the depend file necessary to be checked in? Isn't it created as some part of the build process, like by CMake or something?

Reply:
Hm, no, looks like codeblocks file, same as the other one. Will drop it.